src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -0,0 +1,104 @@
+<template id="limited-restoration-panel">
+  <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
+    <v-radio id="twentyFour-radio" name="selectMonths" label="Two Years" :value=24 />
+    <v-radio id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
+    <v-radio id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
+    <v-radio id="six-radio" name="selectMonths" label="6 months" :value=6 />
+    <v-row class="ml-0">
+      <v-radio id="custom-months" name="selectMonths" label="Month(s): " value="customMonths">
+      </v-radio>
+      <v-text-field class="shrink mx-4" type="number" min=1 max=24 :rules="monthRules" v-model="numberOfMonths"
+        :disabled="!customMonths" @change="onMonthsChanged" filled />
```

Comment:
Try adding prop `dense` to make this component a little shorter.

Also try `hide-details="auto"` to see if that helps.

Also, we've never shown the spin buttons before. You can use `hide-spin-buttons`.

Ref: https://vuetifyjs.com/en/api/v-text-field/

Also, you can try using flexbox to vertically align (center) the radio button and the text field: https://vuetifyjs.com/en/styles/flex/#flex-align

Reply:
It's going to look like this:
![New Design](https://user-images.githubusercontent.com/122301442/217082424-4144f41e-48bf-43a4-82ad-70459f690eb9.PNG)

What do you think?

Reply:
That looks tighter to me (nice). Do the errors still display OK?

Reply:
Yes! The field is reactive so the size is going to change based on the length of the error.

Reply:
Sev I updated the design again. I was checking the UXPin and this is how it looks now. Sorry about that.

![newest design](https://user-images.githubusercontent.com/122301442/217101854-5dd736b6-5567-448b-8107-8372cf42f01d.PNG)

Reply:
That's even better!